pkg/network/netlink/consumer.go

```diff
@@ -337,6 +337,41 @@ func (c *Consumer) dumpTable(family uint8, output chan Event, ns netns.NsHandle)
 	})
 }
 
+// LoadNfConntrackKernelModule requests a dummy connection tuple from netlink conntrack which is discarded but has
+// the side effect of loading the nf_conntrack_netlink module
+func LoadNfConntrackKernelModule(ns netns.NsHandle) error {
+	sock, err := NewSocket(ns)
+	if err != nil {
+		return fmt.Errorf("could not open netlink socket for net ns %d: %w", int(ns), err)
```

Comment:
You want to output the ino for the net ns here, https://github.com/DataDog/datadog-agent/blob/main/pkg/util/kernel/netns.go#L141